etc/php81/php-fpm.d/01-deskpro.conf.tmpl

```diff
@@ -3,7 +3,7 @@ user = dp_app
 group = dp_app
 chdir = /srv/deskpro
 listen = /run/php_fpm_{{.}}.sock
-listen.backlog = 511
+listen.backlog = {{ getenv "PHP_FPM_LISTEN_BACKLOG" "1000" }}
```

Comment:
A higher backlog means more requests will get queued before failing. This means there might be fewer 502's. However, if a request is backlogged long enough that it exceeds the max time, then those 502's may just get converted into 504's instead.

In practice, this change probably only alleviates issues to do with max_children being exceeded in a quick surge, but where the backlog becomes clear relatively quickly.